public/components/context_menu/context_menu.js

```diff
@@ -87,7 +91,7 @@ const generateInContextReport = async (
   };
 
   fetch(
-    `${getApiPath()}/reporting/generateReport?${new URLSearchParams(
+    `../api/reporting/generateReport?${new URLSearchParams(
```

Comment:
There is something wrong here - getApiPath is returning two different things for discover 1.0 and discover 2.0, causing reporting to fail when in new discover, need to look into this